CMakeLists.txt

```diff
@@ -242,11 +242,12 @@ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
 
 # Please do not change compiler/linker flags if You do not know how particular
 # flag is handled by CMake
-set(CMAKE_INSTALL_PREFIX ${CMAKE_CURRENT_BINARY_DIR})
-set(ARCHIVE_OUTPUT_DIRECTORY ./bin)
+#set(CMAKE_INSTALL_PREFIX ${CMAKE_CURRENT_BINARY_DIR})
+set(INSTALL_LIB_DESTINATION ${CMAKE_INSTALL_PREFIX}/lib)
+set(INSTALL_BIN_DESTINATION ${CMAKE_INSTALL_PREFIX}/bin)
 
 set(EXCLUDE_ERROR_FLAGS "-Wno-error=unused-variable -Wno-error=unused-but-set-variable -Wno-error=deprecated-declarations")
-set(CMAKE_CXX_FLAGS "-fPIC -std=gnu++0x -Wall -fvisibility=hidden -Werror ${EXCLUDE_ERROR_FLAGS} -Wuninitialized -Wvla")
```

Comment:
@alexkutsan is it not necessary to handle all warnings for raspberry?

Reply:
New version of compiler on rasbian not compatible with logger and boost with warnings as errors= on